### PR TITLE
add amd ci workflow

### PR DESCRIPTION
**Pull request by Yancey1989**

### CODE REVIEW

**Comment by Yancey1989** [comment]
This PR passed  the  AMD CI job: https://github.com/alibaba/BladeDISC/runs/7037697128?check_suite_focus=true

File: .github/workflows/amd.yml [hunk 1]
```diff
@@ -1,35 +1,38 @@
...
+        git submodule update --depth=1 --init --recursive
+        docker build -t disc-dev-cpu \
+          --build-arg BASEIMAGE=nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04 \
+          --build-arg DISC_HOST_TF_VERSION="tensorflow==1.15.5" \
+          --build-arg DEVICE=cpu \
+          --build-arg HTTPS_PROXY=${HTTPS_PROXY} \
+          -f docker/dev/Dockerfile .
+    - name: Build And Test
+      run: |
+        set -e
```
Comment (fortianyou) [inline .github/workflows/amd.yml:29]: Is that workflow prepared for AMD CPU?

**Review by fortianyou — APPROVED** [review]
LGTM!

File: .github/workflows/amd.yml (hunk 1 above)
Comment (Yancey1989) [reply .github/workflows/amd.yml:29]: Just for an example with CPU, We need a CI script for AMD in the future.

**Pull request merged**